packages/legend-application-query/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -78,6 +78,16 @@ import {
 } from '../stores/postFilterOperators/QueryBuilderPostFilterOperator_In.js';
 import type { QueryBuilderPostFilterOperator } from '../stores/QueryBuilderPostFilterOperator.js';
 
+export class AbortQueryExecutionHelper {
```

Comment:
we really don't need this, also the dependency you introduce is weird since now the state/store depends on something from `components`, let's remove this